src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -981,168 +782,33 @@ def options_request(
         """
         session = self._cache.switch(alias)
         redir = True if allow_redirects is None else allow_redirects
-        response = self._options_request(session, uri, headers, redir, timeout)
+        response = self._common_request(
+            "options",
+            session, 
+            uri, 
+            headers=headers, 
+            allow_redirects=redir, 
+            timeout=timeout)
         logger.info(
             'Options Request using : alias=%s, uri=%s, headers=%s, allow_redirects=%s ' %
             (alias, uri, headers, redir))
 
         return response
 
-    def options(
-            self,
-            alias,
-            uri,
-            headers=None,
-            allow_redirects=None,
-            timeout=None):
-        """ **Deprecated- See Options Request now**
-        
-        Send an OPTIONS request on the session object found using the
-        given `alias`
-
-        ``alias`` that will be used to identify the Session object in the cache
-
-        ``uri`` to send the OPTIONS request to
-        
-        ``allow_redirects`` Boolean. Set to True if POST/PUT/DELETE redirect following is allowed.
-
-        ``headers`` a dictionary of headers to use with the request
-        """
-        logger.warn("Deprecation Warning: Use Options Request in the future")
-        session = self._cache.switch(alias)
-        redir = True if allow_redirects is None else allow_redirects
-        response = self._options_request(session, uri, headers, redir, timeout)
-
-        return response
-
-    def _get_request(
+    def _common_request(
             self,
+            method,
             session,
             uri,
-            params,
-            headers,
-            json,
-            allow_redirects,
-            timeout):
+            **args
+        ):
         self._capture_output()
-
-        resp = session.get(self._get_url(session, uri),
-                           headers=headers,
-                           json=json,
-                           params=self._utf8_urlencode(params),
```

Comment:
seems that you removed some function that were executed before sending data to requests.